### DISCUSSION TITLE
Is it now impossible to skip the victory demo performance?

### DISCUSSION THREAD

**Discussion by zzzz0424** [ikemen-engine/Ikemen-GO#1238] [Q&A] [answered]
In MUGEN, 0.99RC1, and earlier versions, it was possible to skip the victory sequence by pressing any key except the Start button and the directional key. However, in the current nightly build, it is no longer possible to do this (I understand that it is a specification that this is not possible with PotS's character).
Will this be a spec in the next version or later?

**Comment by potsmugen** [comment]
You mean skipping the win poses? I'm not sure I'm following because I can still do that.

**Reply by zzzz0424** [reply]
No, we do not always want to skip it.
There are many characters in MUGEN that display a result screen at the end of the last round.
For example, ikaruga's or ahuron's KOF series characters are implemented on almost every character.
This screen is also shown when a character wins in an arcade match.
It would be depressing to make it impossible to skip that performance at any given moment when you are seeing it for the first time, or when you are continuing over and over again...
If this is to be made a specification, it would be nice if it could be switched in the options.

https://github.com/ikemen-engine/Ikemen-GO/assets/60909119/0a9b70d8-d517-4591-a3b7-7856822d1f7c

**Comment by potsmugen** [comment]
Can you provide some specific details on how to reproduce the issue? I imagine just win a match with Ikaruga's Kyo and try to skip the victory screen? Any difference with Mugen there is unintentional, I think, so this may be a bug.

**Comment by zzzz0424** [comment]
I understand.
First, please download one of the pre-Adelheid characters from the following URL
*The characters after Adelheid are arranged characters that do not exist in KOF98, so there is no victory screen.
http://ikrgmugen.web.fc2.com/kof98.html
Any way you want to do it, just enter any key other than the directional key and the start button when you get to RoundState=4.
Normally you can skip it, but currently nightly you can't.
This is not only ikaruga's creation, but any character has this problem.

**Comment by potsmugen** [comment]
Thanks. But this is weird because it works for me in the 17/05/2023 build:

https://github.com/ikemen-engine/Ikemen-GO/assets/107247004/1fdc5b1c-9884-4f03-a56a-a97a6a562f62

Have you tried in a clean install?

**Comment by zzzz0424** [comment]
I did a clean install and tried it and was able to skip it.
I checked the various files in WinMerge and none of them had any changes, so my nightly may have been out of date.
I apologize for any inconvenience this may have caused.

**Reply by zzzz0424** [reply]
P.S.
I also found that the cause of this is the lifebar I am using.
I can now skip it if I change it back to the default.
I was able to skip even with this lifebar before 0.99RC1, so there may be a problem around the lifebar.
I have included a download link for the lifebar I used for reference.

https://www.mediafire.com/file/m20rwfxogxa76nd/VF5_Lifebars_GO%2528WIP%2529.zip/file

**Comment by potsmugen** [comment, marked as answer]
Seems to be because those lifebars use a different fadeout time. I think one of the next commits may fix this. If not this is indeed a bug.

In the meantime you can edit these lines:
```
fadein.time = 15
fadeout.time = 15
```
To the standard 30.
